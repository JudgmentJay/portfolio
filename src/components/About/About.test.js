import React from 'react'
import renderer from 'react-test-renderer'

import { About } from '../'

it('Renders correctly', () => {
	const tree = renderer
		.create(<About date={new Date()} />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
