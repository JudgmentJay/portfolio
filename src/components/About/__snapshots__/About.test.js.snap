// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<section
  className="section"
  id="about"
>
  <div
    className="container"
  >
    <h3
      className="sectionHeader"
    >
      About Me
    </h3>
    <div
      className="content"
    >
      <div
        className="portrait"
      >
        <picture>
          <source
            media="(max-width: 519px)"
            srcSet="img/portrait_m.webp"
            type="image/webp"
          />
          <source
            media="(max-width: 519px)"
            srcSet="img/portrait_m.jpg"
            type="image/jpeg"
          />
          <source
            media="(max-width: 739px)"
            srcSet="img/portrait_t.webp"
            type="image/webp"
          />
          <source
            media="(max-width: 739px)"
            srcSet="img/portrait_t.jpg"
            type="image/jpeg"
          />
          <source
            srcSet="img/portrait.webp"
            type="image/webp"
          />
          <source
            srcSet="img/portrait.jpg"
            type="image/jpeg"
          />
          <img
            alt="Photo of Jay Posten"
            className=""
            sizes="(max-width: 519px) 460px, (max-width: 739px) 190px, 320px"
            src="img/portrait.jpg"
            srcSet="img/portrait.jpg 460w, img/portrait.jpg 190w, 320w"
          />
        </picture>
      </div>
      <div
        className="intro"
      >
        <p
          className="intro intro--about"
        >
          I am a senior software engineer with over 
          15
           years professional experience and I have a passion for efficient, modular, reusable code. I enjoy creating rich user experiences using modern web technologies such as HTML, CSS/SASS, JavaScript, and React. On the backend I have experience with Node, Express, Next, GraphQL, SQL/SQLite, and MongoDB. I'm highly organized with an eye for user experience and an obsessive attention to detail.
        </p>
        <div
          className="contactInfo"
        >
          <a
            className="contactLink"
            href="tel:3619450085"
          >
            <picture>
              <source
                srcSet="img/icon_phone.webp"
                type="image/webp"
              />
              <source
                srcSet="img/icon_phone.png"
                type="image/png"
              />
              <img
                alt="Phone Icon"
                className=""
                src="img/icon_phone.png"
              />
            </picture>
            <span
              className="contactMethod"
            >
              361.945.0085
            </span>
          </a>
          <a
            className="contactLink"
            href="mailto:jdposten@gmail.com"
          >
            <picture>
              <source
                srcSet="img/icon_mail.webp"
                type="image/webp"
              />
              <source
                srcSet="img/icon_mail.png"
                type="image/png"
              />
              <img
                alt="Email Icon"
                className=""
                src="img/icon_mail.png"
              />
            </picture>
            <span
              className="contactMethod"
            >
              jdposten@gmail.com
            </span>
          </a>
          <div
            className="socialIcons"
          >
            <a
              className="socialIcon"
              href="https://github.com/JudgmentJay"
              rel="noreferrer noopener"
              target="_blank"
            >
              <picture>
                <source
                  srcSet="img/icon_github.webp"
                  type="image/webp"
                />
                <source
                  srcSet="img/icon_github.png"
                  type="image/png"
                />
                <img
                  alt="GitHub Icon"
                  className=""
                  src="img/icon_github.png"
                />
              </picture>
            </a>
            <a
              className="socialIcon"
              href="https://www.linkedin.com/in/jayposten/"
              rel="noreferrer noopener"
              target="_blank"
            >
              <picture>
                <source
                  srcSet="img/icon_linkedin.webp"
                  type="image/webp"
                />
                <source
                  srcSet="img/icon_linkedin.png"
                  type="image/png"
                />
                <img
                  alt="LinkedIn Icon"
                  className=""
                  src="img/icon_linkedin.png"
                />
              </picture>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
